src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.database.Storage;
+import core.basesyntax.service.ReportService;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String HEADER = "fruit,quantity";
+
+    private FruitDao fruitDao;
+
+    public ReportServiceImpl(FruitDao fruitDao) {
+        this.fruitDao = fruitDao;
+    }
+
+    @Override
+    public String create() {
+        String result;
+        if (fruitDao.getAll().isEmpty()) {
+            result = "Storage is empty";
+        } else {
```

Comment:
```suggestion
        if (fruitDao.getAll().isEmpty()) {
            return "Storage is empty";
        }
```

Reply:
Ok, It's just a habbit frob DB development.